torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -534,6 +534,16 @@ static void PrepareListAppendAndInsertForONNX(Block* b) {
   }
 }
 
+static void ChangeInputTypeForInplaceOps(Node* inplaceNode) {
+  // TODO: Check if binary op
```

Comment:
complete the TODO ;)

Reply:
Is the check I added sufficient?

Reply:
try format `node->kind() == aten::add_` etc.